docs/releases/upgrading/upgrade-v11.0.md

```diff
@@ -0,0 +1,70 @@
+# Upgrading from v10 to v11
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v10, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v10, you can gradually upgrade using the existing instructions to v9 and afterwards continue from here.
+
+## Upgrade Steps
+
+!!! warning "Upgrade your invenio-cli"
+
+    Make sure you have the latest `invenio-cli`, for InvenioRDM v11 the release is v1.0.13.
+
+    ```bash
+    $ invenio-cli --version
+    invenio-cli, version 1.0.13
+    ```
+
+For local installations, make sure that you prefix each `invenio` command with `pipenv run`.
+
+### Re-index data
+
+Re-index data in the cluster:
+
+TODO can we use the new command `invenio rdm rebuild-all-indices`?
+```bash
+# recreate indices
+invenio index destroy --yes-i-know
+invenio index init
+
+# reindex records
+invenio rdm-records rebuild-index
```

Comment:
do we need to rebuild indexes here? I do not see anything search related in the following steps. I think it would be ok with only the last `rdm rebuild-all-indices`

Reply:
Thanks for the review and suggestion! I was not sure myself whether it was needed or not.
The upgrade guide is a draft right now, since I am not 100% sure on the steps needed to perform the upgrade. 
I focused more on documenting the configuration variables and changes.